src/promptflow/promptflow/executor/flow_validator.py

```diff
@@ -105,11 +107,19 @@ def _validate_nodes_topology(flow: Flow) -> Flow:
 
     @staticmethod
     def resolve_flow_inputs_type(flow: Flow, inputs: Mapping[str, Any], idx: Optional[int] = None) -> Mapping[str, Any]:
-        """
-        Resolve inputs by type if existing. Ignore missing inputs. This method is used for PRS case
+        """Resolve inputs by type if existing. Ignore missing inputs.
 
-        return:
-            type converted inputs
+        :param flow: The `flow` parameter is of type `Flow` and represents a flow object
+        :type flow: Flow
+        :param inputs: A dictionary containing the input values for the flow. The keys are the names of the
+        flow inputs, and the values are the corresponding input values
```

Comment:
For multi-line strings, the second line and all lines after should indent from the first line:
```python
:param inputs: A dictionary containing the input values for the flow. The keys are the names of the
    flow inputs, and the values are the corresponding input values

```

Reply:
done